projects/cdk/tokens/environment.ts

```diff
@@ -6,9 +6,9 @@ import {TUI_FALSE_HANDLER} from '@taiga-ui/cdk/constants';
 import {fromEvent, map} from 'rxjs';
 
 // https://stackoverflow.com/a/11381730/2706426 http://detectmobilebrowsers.com/
-const firstRegex =
+export const firstRegex =
```

Comment:
We need prefix this since it becomes public. We should probably also group them into 1 somehow.